crates/ruff_python_parser/src/token.rs

```diff
@@ -922,117 +922,116 @@ impl TryFrom<TokenKind> for UnaryOp {
     }
 }
 
-impl fmt::Display for TokenKind {
-    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
-        let value = match self {
-            TokenKind::Unknown => "<Unknown>",
-            TokenKind::StartModule => "StartProgram",
+impl TokenKind {
+    pub(crate) fn display(self) -> &'static str {
```

Comment:
I know you probably changed this because of me, but I think it's fine to implement `Display` for `TokenKind` if we implement it for all variants anyway. I would only recommend using a different method if the signature is different (e.g. it returns `Option<&str>`